src/experimental/safe_html_transform/configure.zcml

```diff
@@ -17,6 +17,15 @@
         provides="Products.GenericSetup.interfaces.EXTENSION"
         />
 
+    <genericsetup:registerProfile
+      name="uninstall"
+      title="experimental.safe_html_transform uninstall"
+      directory="profiles/uninstall"
+      description="Uninstalls the experimental.safe_html_transform package"
+      provides="Products.GenericSetup.interfaces.EXTENSION"
+      />
+
+    
```

Comment:
This is good stuff---plenty of people will be very happy to see you thinking of an uninstall profile already, but there isn't a ``src/experimental/safe_html_transform/profiles/uninstall`` directory just yet. Don't register it until you've got something useful there.